### ISSUE TITLE
Cannot mongodump Because of MeteorToys Collection Names

### ISSUE THREAD

**Issue by abernix** [MeteorToys/allthings#46]
The forward slashes in the collection names are incompatible with the mongodump command on *nix systems.  There is [discussion over at Mongo about disallowing particular characters in collection names](https://jira.mongodb.org/browse/SERVER-4413), and forward slashes are one of them.

The only recommended solution right now is to rename the collection.

```
2015-10-07T14:31:17.688+0300    Failed: error creating bson file
  `../db-backups/10_07_2015_14_19_26/meteor/MeteorToys/Impersonate.bson`:
  open ../db-backups/10_07_2015_14_19_26/meteor/MeteorToys/Impersonate.bson: 
  no such file or directory
```

Luckily this package is `debugOnly` and would never get deployed to production, however this naming convention that MeteorToys uses is currently causing all of my dev dumps to fail.

I've temporarily updated my `mongodump` scripts to include a `--excludeCollectionsWithPrefix=MeteorToys` flag, however this should be fixed asap to avoid anyone with development backups unknowingly not having them done (mine went a week without me noticing, eek!)

**Comment by msavin** [comment]
Thanks for the catch. I'll be sure to update this in the next revision.

**Comment by msavin** [comment]
This issue is resolved the new 2.2.0 release. To update, remove your Meteor Toys package and run `meteor add meteortoys:allthings@2.2.0`. If this issue repeats, or you have other issues, please let me know. :)

**Comment by armandomendivil** [comment]
Thanks! @abernix
